cslabs-backend/Migrations/20191025225433_user_module_table.cs

```diff
@@ -0,0 +1,17 @@
+﻿using Microsoft.EntityFrameworkCore.Migrations;
+
+namespace CSLabsBackend.Migrations
+{
+    public partial class user_module_table : Migration
+    {
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
```

Comment:
Where are the normal builder statements for this table? If you created the table in mysql workbench, you shouldn't because the migration won't see any changes between the model and the database. Make the changes in C#, generate a migration and allow the migration to change the database. If I ran your migration on my side and no `user_modules` table was made.